hiring-developers-with-readthedocs.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: October 30, 2018
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips to Hire Developers with Read the Docs
+==========================================
+
+Read the Docs is probably not the first place you think of if you are recruiting.
+However, over **7 million unique developers use Read the Docs each month** from all over the world.
+We didn't set out to build a better job board,
+but after a number of advertisers used our `ethical ads`_ for recruiting,
+we discovered that Read the Docs was a great place to find developer talent.
+
+.. _ethical ads: https://docs.readthedocs.io/en/latest/advertising/ethical-advertising.html
+
+
+Hiring with Read the Docs works 
+-------------------------------
+
+Developers are not always actively job seeking
+by browsing job boards or company careers pages.
+However, they are on Read the Docs reading about the libraries and frameworks they use.
+Even when people aren't actively looking for a new job,
+many are open to exploring new opportunities.
+
+On Read the Docs, we found that developers engage with hiring ads
+at a higher rate than any other kind of advertising.
```

Comment:
If you were a company that is advertising at RTD with ads that it's about your product, how would you read this sentence? Would it "hurt" you?

Reply:
Anyway, the graphic says exactly the same without words, though.

Reply:
> Would it "hurt" you?

Well it's a double edged sword. For cpc ads it will cost more (we do cpm as well where it wouldn't). However the whole point of ads is for people to engage with them.